pkl-config-java/pkl-config-java.gradle.kts

```diff
@@ -10,14 +10,18 @@ val pklCodegenJava: Configuration by configurations.creating
 val firstPartySourcesJars by configurations.existing
 
 val generateTestConfigClasses by tasks.registering(JavaExec::class) {
-  outputs.dir("build/testConfigClasses")
+  val outputDir = layout.buildDirectory.dir("testConfigClasses")
+  outputs.dir(outputDir)
   inputs.dir("src/test/resources/codegenPkl")
 
   classpath = pklCodegenJava
   mainClass.set("org.pkl.codegen.java.Main")
-  args("--output-dir", "build/testConfigClasses")
-  args("--generate-javadoc")
-  args(fileTree("src/test/resources/codegenPkl"))
+  argumentProviders.add(CommandLineArgumentProvider {
+    listOf(
+      "--output-dir", outputDir.get().asFile.absolutePath,
```

Comment:
It's always safe to use `.path` instead of `.absolutePath` in Gradle build scripts.
If a build script mistakenly creates a relative `File` by using `File("relative/path")` instead of `file("relative/path")`,
`.absolutePath` won't save the day because it will use the wrong base path (current working dir instead of `project.dir`).

Reply:
Will change!